core/deployment/src/main/java/io/quarkus/deployment/builditem/DockerStatusBuildItem.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.deployment.IsDockerWorking;
+
+public final class DockerStatusBuildItem extends SimpleBuildItem {
+
+    private final IsDockerWorking isDockerWorking;
+    private volatile Boolean cachedStatus;
```

Comment:
It does not matter, but just FYI this does not need to be volatile as it is only used within a synchronized method. It only needs to be volatile if you also want to read it outside a sync block.

Reply:
That's right.